repositories/convoy_repository.go

```diff
@@ -27,23 +28,89 @@ func getOwnerId(organizationId string, partnerId null.String) string {
 	return fmt.Sprintf("org:%s", organizationId)
 }
 
+// Ensure event type is included in the event data
+func getEventContent(eventContent models.WebhookEventContent) (string, map[string]interface{}) {
+	eventType := string(eventContent.Type)
+	eventData := eventContent.Data
+	eventData["event_type"] = eventType
+	return eventType, eventData
```

Comment:
so, just to have it all clear in my head: the expectation with convoy (and other solutions) is that we send a map with the event type and id of the object at the root level, along with (optionally) the event body that is nested, and any relevant metadata too if we want ?

Reply:
if all the function does is effectively to add the type to the event data, let's rename it to something along the lines of "addEventTypeToData" and make it return just the map ? (and use the original webhookContent type where we need it)